src/templates/solsys_code_observatory/observatory_create.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'tom_common/base.html' %}
+{% load bootstrap4 %}
+{% block title %}Create Observatory from MPC Code{% endblock %}
+
+{% block content %}
+<h4>Create a new Observatory in FOMO from MPC code. </h4>
```

Comment:
Include a link to https://www.minorplanetcenter.net/iau/lists/ObsCodesF.html here.